rktlet/runtime/helpers.go

```diff
@@ -293,3 +293,28 @@ func getIP(networks []netinfo.NetInfo) string {
 	}
 	return foundIP
 }
+
+// passFilter returns true if the container satisfies the filter,
+// which means it will be return.
+// Otherwise, returns false.
```

Comment:
This looks like a convoluted way of saying "whether the target container satisfies the filter."